tests/cli/test_cmds_status.py

```diff
@@ -563,6 +561,115 @@ def test_apply_args_filters_clusters_and_instances(
         "instance3": mock_inst3.__class__,
     }
 
+@patch("paasta_tools.cli.cmds.status.get_instance_configs_for_service", autospec=True)
+@patch("paasta_tools.cli.cmds.status.list_services", autospec=True)
+@patch("paasta_tools.cli.cmds.status.figure_out_service_name", autospec=True)
+@patch("paasta_tools.cli.cmds.status.validate_service_name", autospec=True)
+def test_apply_args_filters_shorthand_notation(
```

Comment:
You can reduce the number of test functions by using pytest parameterization. Here's the documentation: https://docs.pytest.org/en/latest/parametrize.html. Specifically, you can parameterize over the service name and expected result.